importmap/jinja2/extension.py

```diff
@@ -0,0 +1,32 @@
+from jinja2 import nodes
+from jinja2.ext import Extension
+
+from importmap.templatetags.importmap import importmap_scripts
+
+
+SCRIPT_TEMPLATE = """
```

Comment:
Do you know how to put this in a template file (that can be overridden by a user)? Would it be `templates/jinja2/importmap/scripts.html`? https://docs.djangoproject.com/en/4.1/topics/templates/#module-django.template.loader

May be overkill, but I'd like it in a place that can be overridden for now because I haven't done anything with settings around the shim (version, whether or not to include it).

Reply:
Ah, I was wondering why a template file was used for the scripts. Didn't think about the fact that it allows users to override it, but it makes sense.

However, from the top of my head I can't think of a simple way to achieve the same thing with Jinja.

Reply:
One possible solution that I can think of would be to use *Django's* templating system to render the scripts-snippet. Changing the render function to something like:

```python
from django.template import Engine, Context

    ...
    def _render(self, _):
        template = Engine.get_default().get_template("importmap/scripts.html")
        return template.render(Context(importmap_scripts()))
```

That should allow users to override the file, but maybe a bit strange that one would use a Django template for that when using the Jinja extension?